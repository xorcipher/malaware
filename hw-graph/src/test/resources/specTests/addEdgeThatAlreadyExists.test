# Tests that an edge can be added to two nodes within a graph when the edge already exists.

# Creates a new edge between two nodes.
CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B edgeA
AddEdge graph A B edgeB