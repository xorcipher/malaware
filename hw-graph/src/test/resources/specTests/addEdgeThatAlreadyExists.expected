# Tests that an edge can be added to two nodes within a graph when the edge already exists.

# Creates a new edge between two nodes.
created graph graph

added node A to graph
added node B to graph

added edge edgeA from A to B in graph
added edge edgeB from A to B in graph